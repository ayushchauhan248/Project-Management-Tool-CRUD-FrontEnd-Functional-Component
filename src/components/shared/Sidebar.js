import "./Sidebar.css";
import { Link } from "react-router-dom";
import { MdCreateNewFolder } from "react-icons/md";
import { ImHome } from "react-icons/im";
import { RiLogoutBoxFill } from "react-icons/ri";
import React, { memo } from "react";

const Sidebar = (props) => {
  return (
    <div id="sidebar">
      <div id="uul">
        <Link to="/dashboard" id="llnk" className="doFlex">
          <ImHome id="llii"></ImHome>
          <div id="lli">DASHBOARD</div>
        </Link>

        <Link to="/create" id="llnk" className="doFlex">
          <MdCreateNewFolder id="llii"></MdCreateNewFolder>
          <div id="lli">CREATE</div>
        </Link>

        <Link to="/" id="llnk" className="doFlex">
          <RiLogoutBoxFill id="llii"></RiLogoutBoxFill>
          <div id="lli">LOGOUT</div>
        </Link>
      </div>
    </div>
  );
};
export default memo(Sidebar);
