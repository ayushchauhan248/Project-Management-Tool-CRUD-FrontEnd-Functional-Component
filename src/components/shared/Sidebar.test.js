import { shallow } from "enzyme";
import React from "react";
import Sidebar from "./Sidebar";

it("should render snapshot of navbar", () => {
  const component = shallow(<Sidebar />);
  expect(component).toMatchSnapshot();
});
