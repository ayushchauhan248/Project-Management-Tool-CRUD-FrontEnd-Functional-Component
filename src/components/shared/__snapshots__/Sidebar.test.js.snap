// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render snapshot of navbar 1`] = `
<div
  id="sidebar"
>
  <div
    id="uul"
  >
    <Link
      className="doFlex"
      id="llnk"
      to="/dashboard"
    >
      <ImHome
        id="llii"
      />
      <div
        id="lli"
      >
        DASHBOARD
      </div>
    </Link>
    <Link
      className="doFlex"
      id="llnk"
      to="/create"
    >
      <MdCreateNewFolder
        id="llii"
      />
      <div
        id="lli"
      >
        CREATE
      </div>
    </Link>
    <Link
      className="doFlex"
      id="llnk"
      to="/"
    >
      <RiLogoutBoxFill
        id="llii"
      />
      <div
        id="lli"
      >
        LOGOUT
      </div>
    </Link>
  </div>
</div>
`;
